Backend/WebApi-PointMap/Controllers/UserManagementController.cs

```diff
@@ -41,24 +43,30 @@ public IHttpActionResult GetAllUsers()
                     else
                     {
                         _db.SaveChanges(); // save updated user session
-                        throw new UserIsNotAdministratorException("Non-administrators cannot delete users.");
+                        throw new UserIsNotAdministratorException("Non-administrators cannot view all users.");

```

Comment:
Same here as above, since we expect the user to either have autho or not, if not just return response, no need to thow/catch exception.